<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Map</class>
 <widget class="QMainWindow" name="Map">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1850</width>
    <height>950</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QPushButton" name="InButton">
    <property name="geometry">
     <rect>
      <x>1650</x>
      <y>0</y>
      <width>93</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(160, 255, 143);
font: 75 10pt &quot;Consolas&quot;;</string>
    </property>
    <property name="text">
     <string>放大 +</string>
    </property>
   </widget>
   <widget class="QPushButton" name="outButton">
    <property name="geometry">
     <rect>
      <x>1750</x>
      <y>0</y>
      <width>93</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(160, 255, 143);
font: 75 10pt &quot;Consolas&quot;;</string>
    </property>
    <property name="text">
     <string>缩小 -</string>
    </property>
   </widget>
   <widget class="QComboBox" name="startBox">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>0</y>
      <width>141</width>
      <height>41</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>0</y>
      <width>81</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(0, 0, 0);
color: rgb(255, 255, 255);
font: 75 10pt &quot;Consolas&quot;;</string>
    </property>
    <property name="text">
     <string> 起始点：</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>310</x>
      <y>0</y>
      <width>81</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(0, 0, 0);
font: 75 10pt &quot;Consolas&quot;;
color: rgb(255, 255, 255);</string>
    </property>
    <property name="text">
     <string> 终点：</string>
    </property>
   </widget>
   <widget class="QComboBox" name="endBox">
    <property name="geometry">
     <rect>
      <x>390</x>
      <y>0</y>
      <width>141</width>
      <height>41</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>930</x>
      <y>0</y>
      <width>101</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(85, 170, 127);
font: 75 10pt &quot;Consolas&quot;;</string>
    </property>
    <property name="text">
     <string>最短距离为：</string>
    </property>
   </widget>
   <widget class="QPushButton" name="shortBtn">
    <property name="geometry">
     <rect>
      <x>630</x>
      <y>0</y>
      <width>151</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(85, 170, 255);
font: 75 10pt &quot;Consolas&quot;;</string>
    </property>
    <property name="text">
     <string>开始查找最短路径</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="showDist">
    <property name="geometry">
     <rect>
      <x>1030</x>
      <y>0</y>
      <width>131</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(0, 0, 0);
font: 75 10pt &quot;Consolas&quot;;</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLineEdit" name="edit_search">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>70</y>
      <width>891</width>
      <height>61</height>
     </rect>
    </property>
   </widget>
   <widget class="QWidget" name="mapWidget" native="true">
    <property name="geometry">
     <rect>
      <x>380</x>
      <y>260</y>
      <width>831</width>
      <height>361</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="reduceBtn">
    <property name="geometry">
     <rect>
      <x>1330</x>
      <y>70</y>
      <width>89</width>
      <height>61</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="searchBtn">
    <property name="geometry">
     <rect>
      <x>980</x>
      <y>70</y>
      <width>131</width>
      <height>61</height>
     </rect>
    </property>
    <property name="text">
     <string>搜索</string>
    </property>
   </widget>
   <widget class="QPushButton" name="enlargeBtn">
    <property name="geometry">
     <rect>
      <x>1170</x>
      <y>70</y>
      <width>89</width>
      <height>61</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="piclabel">
    <property name="geometry">
     <rect>
      <x>380</x>
      <y>260</y>
      <width>831</width>
      <height>361</height>
     </rect>
    </property>
    <property name="text">
     <string>l</string>
    </property>
   </widget>
   <widget class="QGraphicsView" name="graphicsView">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>-150</y>
      <width>1471</width>
      <height>861</height>
     </rect>
    </property>
    <property name="verticalScrollBarPolicy">
     <enum>Qt::ScrollBarPolicy::ScrollBarAsNeeded</enum>
    </property>
    <property name="horizontalScrollBarPolicy">
     <enum>Qt::ScrollBarPolicy::ScrollBarAsNeeded</enum>
    </property>
   </widget>
   <widget class="QWidget" name="gridLayoutWidget">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>-140</y>
      <width>160</width>
      <height>80</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="gridLayout"/>
   </widget>
   <widget class="QPushButton" name="btnClose">
    <property name="geometry">
     <rect>
      <x>1230</x>
      <y>0</y>
      <width>121</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>返回</string>
    </property>
   </widget>
   <zorder>mapWidget</zorder>
   <zorder>piclabel</zorder>
   <zorder>InButton</zorder>
   <zorder>outButton</zorder>
   <zorder>graphicsView</zorder>
   <zorder>edit_search</zorder>
   <zorder>gridLayoutWidget</zorder>
   <zorder>searchBtn</zorder>
   <zorder>enlargeBtn</zorder>
   <zorder>reduceBtn</zorder>
   <zorder>label</zorder>
   <zorder>label_2</zorder>
   <zorder>label_3</zorder>
   <zorder>startBox</zorder>
   <zorder>showDist</zorder>
   <zorder>shortBtn</zorder>
   <zorder>endBox</zorder>
   <zorder>btnClose</zorder>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1850</width>
     <height>27</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
